reactioncommerce/reaction-component-library
# feat: AddressReview component

Resolves #122 & #277 
Impact: **major**  
Type: **feature**

## Component
**AddressReview**
  *  Based on this [spec](https://github.com/reactioncommerce/reaction-component-library/issues/122)

**Address**
  *  Created an Address component that displays an Address

**SelectableList Enhancement** 
  *   Adds `isHorizontal` prop: Updated to allow a horizontal layout along with the vertical style.
  *   Adds `isStacked` prop: Updated to allow `detail` to display stacked under the label along with the right/left aligned.

## Screenshots
<img width="1065" alt="screen shot 2018-10-18 at 1 48 41 pm" src="https://user-images.githubusercontent.com/3673236/47183317-9046d780-d2dc-11e8-9c7e-b63efa19e8c7.png">
⚠️ The alert is not part of this component. The alert will be worked on next. This alert is just a mock-up.

## Breaking changes
N/A

## Testing
1. **AddressReview**: https://deploy-preview-336--stoic-hodgkin-c0179e.netlify.com/#!/AddressReview
    * Open broswer console and test the example implamentation towards the bottom of the page.
    * You should see the selected address object in the console after you click submit.
2. **Address**: https://deploy-preview-336--stoic-hodgkin-c0179e.netlify.com/#!/Address
    * Verify it's workings and looks as expected.
3. **SelectableList**: https://deploy-preview-336--stoic-hodgkin-c0179e.netlify.com/#!/SelectableList
    *  Verify the `isHorizontal` prop is working and looks as expected. 
3. **SelectableItem**: https://deploy-preview-336--stoic-hodgkin-c0179e.netlify.com/#!/SelectableItem 
    * Verify the `isStacked` prop is working and looks as expected. (see Address as detail header)
